requirements.txt

```diff
@@ -7,6 +7,7 @@ flake8==3.5.0
 gunicorn==19.8.1
 idna==2.6
 mccabe==0.6.1
+Pillow==5.1.0
```

Comment:
Any reason for adding this?
I don't think we need this since we are only dealing with the URL of image